### DISCUSSION TITLE
Can't resume download

### DISCUSSION THREAD

**Discussion by ShahriarKh** [beekeeper-studio/beekeeper-studio#967] [Q&A]
![image](https://user-images.githubusercontent.com/31452340/148652634-51188b11-07d6-4d88-befc-6a97270a1373.png)
The file started downloading without problems, but near 95%, it stopped and is asking for credentials!

**Comment by rathboma** [comment]
Huh. That's super weird. Are you on a corporate proxy or something like that?

**Reply by ShahriarKh** [reply]
Nope. Anyways, I just completely restarted the download and it worked.
Also, here's the reason: https://stackoverflow.com/questions/70634546/authorization-required-when-downloading-from-objects-githubusercontent-com
